scripts/genproto.sh

```diff
@@ -16,9 +16,14 @@ if ! [[ $(protoc --version) =~ "3.0.0" ]]; then
 fi
 
 export PATH=.:${PATH}
+cd $GOPATH/src/github.com/coreos/rkt
```

Comment:
To add a short answer: Unfortunately you have to have a `GOPATH` to build `protoc-gen-go`.